docs/data/upload.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Upload Data"
+linkTitle: "Upload Data"
+description: "Upload data to the Viam app from your local computer or mobile device using the data client API, Viam CLI, or Viam mobile app."
+weight: 40
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+image: "/services/icons/data-capture.svg"
+imageAlt: "Upload data to the Viam app"
+images: ["/services/icons/data-capture.svg"]
+# SME: Alexa Greenberg
+---
+
+If you configured [data capture](/data/capture/) on your machine, data is automatically uploaded to the Viam cloud from the directory and at the interval you specified.
+However, if you want to upload a batch of data manually from somewhere else either on your machine, or from your personal computer or mobile device, you have several options:
```

Comment:
Supernit: think this is the correct form:

```suggestion
However, if you want to upload a batch of data manually from somewhere else, either on your machine or from your personal computer or mobile device, you have several options:
```

Reply:
Agree there should be a comma after "else," thank you! Methinks a comma could also come after machine, but perhaps that one could go either way. *hmm parrot*